deploy/cluster_role_kubevirt.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: benchmark-operator
```

Comment:
we already have a ClusterRole with the name `benchmark-operator` at https://github.com/cloud-bulldozer/honeybadger/blob/master/deploy/cluster_role.yaml not sure how it'd work with 2 clusterrole sharing same names.